src/auth/components/Login.jsx

```diff
@@ -37,3 +63,16 @@ export default class Login extends Component {
 Login.propTypes = {
   login: PropTypes.func.isRequired,
 };
+
+
+function mapStateToProps(state) {
+  return {
+    errorMessage: state.errorReducer
+  };
+}
+
+
+export default connect(mapStateToProps, {resetErrorMessage})(Login);
```

Comment:
Please don't connect error stuff in Login component, you can move it to Auth.jsx component.